www/install.rst

```diff
@@ -79,14 +75,32 @@ popular package managers you can install.
 Macports
 ~~~~~~~~
 
-To install the scipy stack for Python 2.7 with `Macports
+To install the SciPy stack for Python 2.7 with `Macports
 <http://www.macports.org>`_ execute this command in a terminal::
 
     sudo port install py27-numpy py27-scipy py27-matplotlib py27-ipython +notebook py27-pandas py27-sympy py27-nose
 
+Homebrew
+~~~~~~~~
```

Comment:
Hmmm...boldface makes the section headers a little too small on my computer for my liking.  Their size starts to become indistinguishable from that of the section text.